### ISSUE TITLE
Feature request: Clickable disguise command.

### ISSUE THREAD

**Issue by andrewkm** [libraryaddict/LibsDisguises#532]
Several plugins (such as worldguard for example) have clickable chat commands that make life easier.
It would be nice if the /disguise command allowed the same:

For example, clicking minecart_chest would input the following into the chat:
https://i.imgur.com/ZZRPWZw.png

Would be a really nice quality of life thing when exploring all the various disguises a user has access to. :)

**Comment by libraryaddict** [comment]
I agree, but the biggest holdup is that it'd require me to rewrite the entire message system into an easier one that would also be more complicated, especially since I do translation supports..

The other biggest holdup is that I'm working at CubeCraft which has its own projects to do :\

**Comment by libraryaddict** [comment]
Semi-support in the current dev build and next release, using mini-message.
I say semi because it'll probably break if/when I add actual fancy commands.

The reason behind the holdup is because I don't really want to limit what you can display, to remain compatibility with older stuff, and because of stuff required.

**Comment by libraryaddict** [comment]
https://docs.adventure.kyori.net/minimessage.html#format

**Comment by andrewkm** [comment]
Not really sure what is meant by semi-supported, been trying everything, can't seem to get it to work.
Maybe I'm just not trying the correct way. :(

**Comment by andrewkm** [comment]
@libraryaddict how would I get it to work in the current "semi-supported" state?

**Comment by libraryaddict** [comment]
Yeah, I need to do more work on this.

Did some thinking last night, and frankly the minimessage stuff still has some issues here and there. Sometimes the command output seems to be doing the raw text.

As for the fancy command stuff, I think it is "supported" but not really. Supported in that you can do fancy stuff, not really in that it needs placeholders or something.

I can't honestly give you any reassurances sorry.
